server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -71,6 +73,13 @@ export class ScriptoriumLambda implements IPartitionLambda {
             return;
         }
 
+        let metric: Lumber<LumberEventName.ScriptoriumHandler>;
```

Comment:
I am a little concerned about how noisy this might be. But then again, sampling might make the metric unhelpful. I wonder if we should keep this turned off by default and turn in on only for investigations.

Reply:
Hey @pradeepvairamani , if we turn this on only for investigations after an issue happened, we would have already lost the telemetry. We would need to have this on so that its captured and we can investigate if or when an issue  occur. 
Apart from issues investigation, I am also hoping to get some insights into the general behavior of scriptorium processing a batch of messages (eg: latency).

Let me know if you are strongly concerned about its noisiness and we can discuss if enabling it in certain clusters (and not all) might help. :)

Reply:
Sure. Thanks @shubhi1092. I think it might be a good idea to start with one cluster. But this is something that we do not need to discuss in this PR. We can think through it when we are consuming this in AFR.